templates/distribution/manifests/dr/secrets/rclone.conf.tpl

```diff
@@ -0,0 +1,13 @@
+# Copyright (c) 2017-present SIGHUP s.r.l All rights reserved.
+# Use of this source code is governed by a BSD-style
+# license that can be found in the LICENSE file.
+
+{{- if eq .spec.distribution.modules.dr.etcdBackup.type "all" "s3" }}
+[minio]
```

Comment:
let's call this `s3` instead of `minio`, the name of the target storage should generic, it could be that the user does not uses minio

Reply:
It's [here](https://rclone.org/s3/#minio), but I agree with you, we should use the S3 main provider.